_includes/wiki/en/footertext.md

```diff
@@ -3,4 +3,4 @@
 **Tell us about Jamulus - [take our anonymous survey!](https://forms.gle/hSSjsxjWj2Pnp5kr7)**
 
 
-_This documentation is licensed under a [Creative Commons License](https://creativecommons.org/licenses/by-sa/4.0/deed.en){: target="_blank" rel="noreferrer" }. Want to [contribute](Contribution)?_
+_This documentation is licensed under a [Creative Commons Licence](https://creativecommons.org/licenses/by-sa/4.0/deed.en){: target="_blank" rel="noreferrer" }. Want to [get involved](Contribution)?_
```

Comment:
Um, but CC spell the noun "license" in the URL.

Reply:
That's because they want to be Americans, but we decided to use British English as standard for our content :-)

Reply:
Also, `https://creativecommons.org/licenses/by-sa/4.0/` seems to be the more canonical link.